src/shared/tpm2-util.c

```diff
@@ -1753,12 +1774,13 @@ static int tpm2_make_policy_session(
                                        "Failed to open session in TPM: %s", sym_Tss2_RC_Decode(rc));
 
         if (pubkey_pcr_mask != 0) {
-#if HAVE_OPENSSL
                 log_debug("Configuring public key based PCR policy.");
 
-                /* First: load public key into the TPM */
+                /* Convert the PEM key to TPM2 format */
+                _cleanup_free_ void *fp = NULL;
```

Comment:
looks a bit weird to have the block declarations below the log, maybe invert them

Reply:
ack, updated, thnx